### ISSUE TITLE
YouTube Plus taken down from Chrome Webstore (no reason has been given)

### ISSUE THREAD

**Issue by ParticleCore** [ParticleCore/Particle#483]
As of today the extension is no longer available in the Chrome Webstore because the Google Team has taken it down. I have not received any emails warning beforehand or explaining this decision and I am having a hard time trying to contact them directly.

Until I get any kind of response from the Webstore there is nothing I can do except telling Chrome users to use the userscript version instead, together with the Tampermokey extension (or similar userscript manager).

I apologize for this situation, but it is something out of my control.

**Comment by SupDos** [comment]
I noticed that the extension had stopped working, so I uninstalled it and went to install it only to find it was taken down.  Plus the extension folder has been removed too so I've lost all my settings! This sucks. Really hope you get a response soon. Thanks for the extension!

**Comment by ParticleCore** [comment]
The Webstore has been contacted 3 times. An explanation has been requested, but I am still not receiving anything back. The only answer I received so far was a generic message stating that the "pertinent" team would communicate with me. It has been 3 days and no communication has taken place.

I have also renamed the extension to "Particle for YouTube" and still was rejected after submitting it for manual review. I am still unaware as to why it was taken down and why it keeps staying down, the Webstore team is not giving any explanations whatsoever.

**Comment by ParticleCore** [comment]
Today will be the last day I will wait for any update on this situation, I have not received once since it all started.

If this is not resolved soon I am considering moving the Chrome extension support to Opera. Chrome users will always be able to use the userscript version with a userscript manager extension, but no support would be available to any Chrome specific problems.

**Comment by NovaViper** [comment]
Well that's Google for you, just take down good extensions/apps and never give an explanation for it.. Google's extremely notorious for that.

**Comment by ParticleCore** [comment]
I have never received any updated regarding this situation and as a result of this I will [move to the Opera browser](https://github.com/ParticleCore/Particle/wiki/Download#opera). Chrome support will continue to exist until the migration is complete.

**Comment by NovaViper** [comment]
Hey, would it be possible to add in Viviadli support?

**Comment by ParticleCore** [comment]
@NovaViper No, it would only be Opera or Vivaldi, more than 2 browsers is too much for me for now, but if someone else wants to fork and support it then by all means go for it as long as the short license is respected.

**Comment by NovaViper** [comment]
@ParticleCore Awh ok.. Though, I would think the Opera extension would work on Vivaldi since they're both based on Chromium (in theory that is)

**Comment by ParticleCore** [comment]
It is very possible that it is compatible, but if there are any Vivaldi only bugs I will not be providing support, that's just what I mean by it.

**Comment by NovaViper** [comment]
Oh ok

**Comment by NovaViper** [comment]
Hey I just tested it on Vivaldi and it works perfectly!

**Comment by erkinalp** [comment]
Why not vanilla Chromium? Then all Chrome forks may benefit.

**Comment by ParticleCore** [comment]
@erkinalp Not having an extension store was a decisive factor.

**Comment by survivor303** [comment]
I write a little blog post about google's new shit what they doing to their users and its contect creators, especially for @ParticleCore and YouTube+ : http://www.granstrom.fi/2017/04/olenko-saanut-tarpeekseni-googlen.html that is in Finnish language but perhaps some translators can change it to english ;)

**Comment by q1k** [comment]
I just wanna say that, any Chrome or Opera extension can be installed on either of the browsers. For Opera there is an extension that will allow you to install extensions from the chrome webstore.

As for Chrome, well it's a bit trickier. You will have to download the Opera extension, then unpack it with any archiver like 7zip. Next, enter developer mode in the extensions manager and "load unpacked extension". Just be aware that, updating won't work, you will have to manually do this with every new version.

So to avoid all this hastle, just get the userscript. Install tampermonkey and then the userscript version.

**Comment by killown** [comment]
I will stop using chrome or chromium browser right now, this shit makes me angry, wtf, they remove such a useful extension and we know why they did it, because the blacklist channel feature... they are evil we should avoid their products.

**Comment by NovaViper** [comment]
@killown Hey you should look at Opera or Vivaldi, I've tried both and they are pretty good. I couldn't use Opera because it doesn't have Join, and I could use Vivaldi because it didn't have a mobile browser (but will soon get one)

**Comment by killown** [comment]
@NovaViper firefox is unusable right now, scroll bar lag makes is so annoying, I tried vivaldi and I liked it

**Comment by NovaViper** [comment]
@killown Yeah.. I've had to dump Firefox about 3 times before finally settling with Chrome.. But ever since I tried Vivaldi, I really might leave Chrome for good and stick with Vivaldi (that tab stacking feature is awesome ^^) but the deal breaker for me as of now is that Vivaldi doesn't have a mobile app browser.

**Comment by BooBerry** [comment]
My best guess for the reason why they removed it is because of the **Disable advertisements outside the video page**, **Disable advertisements in the video page** and **Enable advertisements only in videos from subscribed channels** features. In the past, they've been very touchy in regards to extensions that remove/bypass advertisements on YouTube (even though it seems there are a few extensions still available that do this - I'm guessing it's only a matter of time they're removed). If you were able to complete #351 you *may* be able to resubmit the extension back to the Web Store, and they may not remove it this time.

**Comment by ParticleCore** [comment]
I waited one week for them to tell me why the extension was removed. During that time I posted in their forums and I contacted them multiple times, they never answered. They also failed to state the take down reason, which is something they wrote in their documentation that they do when they take down extensions.

I changed the name to just Particle and applied for review, it was rejected.

I removed the icon and applied for review, it was rejected.

Again, not once they sent an explanation for why it was rejected.

The reason cannot be because of removing ads, otherwise other marginally more popular extensions that have been on the webstore for far longer with the same capability would have been already taken down as well (Magic Actions, for example). Even if it were, I will not be hacking away features from the extension and try to re-submit it for review over and over and over again until they approve it without having them telling me exactly why the extension was taken down. The guessing games are over for me.

Google made their decision and they did not show a slim of respect by explaining why the extension was taken down and I have already tried correcting whatever it was that needed to be corrected on my own initiative. This extension will not return to the webstore and support for Chrome browsers will no longer exist after the migration is completed, and this decision is final.

On the positive side as a result of this situation - since before this I was not interested in adding some special features because I knew they would most certainly result in being banned from the webstore - now I no longer have those restrictions and I am free to add those features. This means that features such as video download are back on the table.

**Comment by NovaViper** [comment]
@ParticleCore Google is a pain in the butt.. but what browser do you plan on moving to? Vivaldi? (Somewhat Chromium based but can add Opera extensions) Opera? Maybe Firefox? (Though that browser atm is having major problems with their extensions as they are moving to WebExtensions [Where MANY addons that modified Firefox will die by Firefox 57 because of major compatibility changes]... I wouldn't recommend it >_>) Or maybe even Microsoft Edge?

**Comment by ParticleCore** [comment]
@NovaViper That decision has already been announced here: https://github.com/ParticleCore/Particle/issues/485

**Comment by NovaViper** [comment]
@ParticleCore Oh ok. I was just curious on where your moving the extension to. Hm.. would this completely rule out the possibility for support with Vivaldi? (I'm just wondering before I completely move from Chrome to either Opera or Vivaldi as I would like to know if this extension would get support on either one of these browsers -  particularly Vivaldi - in case I have any problems)

**Comment by ParticleCore** [comment]
Support will only be available for Firefox and Opera. Vivaldi is not included.

**Comment by NovaViper** [comment]
@ParticleCore Awh that sucks.. maybe someone will come by and pick up support for Vivaldi

**Comment by dyingjedi** [comment]
You all should be using Adguard. Run it in proxy mode for your whole in home network etc. then you do not need tampermonkey or the likes on the browsers themselves anylonger!

![](https://lh3.googleusercontent.com/-SlL_cUVq9lI/WQdFJW79DHI/AAAAAAAACfI/PUCRcI7X9mEiC2MXTmDKboeiCgydVUUbQCL0B/h600/2017-05-01.png)

**Comment by ParticleCore** [comment]
@dyingjedi Thanks for that share, I had no idea AdGuard had come so far, but keep in mind that I am not making sure the extension works correctly with AdGuard, so if anything breaks it falls on their team to fix it if they think it is worth fixing.

**Comment by alphapapa** [comment]
@dyingjedi 

> You all should be using Adguard. Run it in proxy mode for your whole in home network etc. then you do not need tampermonkey or the likes on the browsers themselves anylonger!

Do you ever take your laptop away from home?

**Comment by LB--** [comment]
Suddenly the extension seems to have been restored to the webstore and automatically downloaded and installed to my Chrome browsers:
https://chrome.google.com/webstore/detail/youtube-plus/nkdalpbojfdilmiboaiedicdbigdabpb

**Comment by ParticleCore** [comment]
@LB--  I just got the email from the Webstore team stating that the email explaining the take down should have been sent, but since I made the appropriate changes the extension has been put back on.

It appears that I have violated their branding guidelines, which I suspected to be the YouTube at the beginning of the extension name and/or the icon itself. I don't know why they have restored the version with intact name and icon, but I will try to update it and see if it stays on.

If it does stay on then I guess I'll have to support 3 browsers...

**Comment by ParticleCore** [comment]
It appears the Webstore has restored the extension, I will see if it remains up and if it does then I will not stop Chrome support.

**Comment by Tim-Van-C** [comment]
Can someone explain how you use Tampermonkey with userscripts? I want to use the extension in chrome but I'm new to all of this. 

I just want one feature, to stop autoplay in youtube playlists... Because of the new youtube layout the other extension I used (youtube next-video stopper) doesn't work anymore.

**Comment by LB--** [comment]
@Tim-Van-C this extension works with the new YouTube layout in Chrome: https://chrome.google.com/webstore/detail/disable-html5-autoplay/efdhoaajjjgckpbkoglidkeendpkolai

**Comment by SharpOB** [comment]
Vivaldi works 100% with the Chrome version of the addon. If Opera does too, then it should be easy to support both at the same time.

**Comment by gappleto97** [comment]
Will you continue to support Firefox?

**Comment by ParticleCore** [comment]
@gappleto97 Until the new layout becomes default I will still be supporting the same browsers, that hasn't changed.

**Comment by triune** [comment]
@ParticleCore as to why your extension was taken down with no explanation... ever think maybe that it's because you're using the term YouTube that is trademarked by Google? The Chrome Web Store rules do say to avoid doing this specifically: https://developer.chrome.com/webstore/program_policies

Of course, this is probably the other reason why:
http://www.androidpolice.com/2017/09/22/youtube-v12-37-hints-new-subscription-tier-called-youtube-plus-apk-teardown/

**Comment by ParticleCore** [comment]
@triune perhaps maybe try reading the replies before posting the next time:

1 - This happened half a year ago already  

> ParticleCore commented on Apr 5 • edited 

https://github.com/ParticleCore/Particle/issues/483#issue-219478719

2 - I already explained the probable causes, which I was already aware of before, still they restored it with no changes made to either the icon or naming

>It appears that I have violated their branding guidelines, which I suspected to be the YouTube at the beginning of the extension name and/or the icon itself. I don't know why they have restored the version with intact name and icon, but I will try to update it and see if it stays on.

https://github.com/ParticleCore/Particle/issues/483#issuecomment-303818374

3 - The official YouTube Plus "feature" was only included in the APK around monday last week (I decompiled it), meaning that this wasn't publicly known 6 months ago. However, there is no longer a problem since I sold the Chrome extension version and it was taken down shortly after with the new owner.

Either way, it is no longer my problem, but I am considering changing the name of the extension on the AMO store just to make it less of an headache for future users of the upcoming official service.
